packages/user-interface/src/tests/pages/CasesPage.test.tsx

```diff
@@ -28,8 +24,7 @@ describe("The CasesPage", () => {
     expect(tabAfgerondeZaken().getAttribute("aria-selected")).toBe("false");
 
     expect(openZaak1()).toBeVisible();
-    let screenDate = new Date(Date.parse(openZaak1Date));
-    expect(screen.getByText(screenDate.toLocaleDateString())).toBeVisible();
+    expect(screen.getByText("1 januari 2024")).toBeVisible();
```

Comment:
Werkt dit ook op Engelstalige omgevingen? Ik had hier eerder namelijk issues mee dat tests bij mij faalden omdat mijn OS/browser Engels is.

Reply:
Ik heb daar zelf lokaal geen issue mee gehad en mijn systeem staat ook op engels. Ook de Github actions is alles netjes geslaagd. Staat de local niet standaard gewoon op NL